GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java

```diff
@@ -689,9 +696,18 @@ public boolean addSelectableObject(GVRSceneObject object) {
         }
         object.setSensor(cursorSensor);
         object.getEventReceiver().addListener(cursorSensor);
+        scene.addSceneObject(object);
```

Comment:
Is this line required? If yes then you might be doing scene.addSceneObject twice in the 3d-cursor sample.